.travis.yml

```diff
@@ -11,8 +11,8 @@ env:
 install:
   - pip install -U pip setuptools
   - '[[ "$DEPS" == "devel" ]] && pip install git+https://github.com/pallets/flask.git || :'
-  - pip install -U pytest pytest-cov pytest-mock pytest-pep8 coveralls
-  - '[[ "$TRAVIS_PYTHON_VERSION" == "2.7" ]] && pip install flask-wtf python-ldap || :'
+  - pip install -U pytest pytest-cov pytest-mock pytest-pep8 coveralls python-ldap
+  - '[[ "$TRAVIS_PYTHON_VERSION" == "2.7" ]] && pip install flask-wtf || :'
```

Comment:
I think you need to move flask-wtf out of this if (which can then go away) as well